### PR TITLE
Switched to on-line std and mean estimation for window.

### PR DESCRIPTION
**Pull request by spolakh**
Benchmarking shows 3x speedup comparing to the previous approach (moreover, it also address potential floating-point precision issues as noted e.g. here: http://dsp.stackexchange.com/questions/811/determining-the-mean-and-standard-deviation-in-real-time)

### CODE REVIEW

File: src/symtseries.c [hunk 1]
```diff
@@ -215,72 +215,49 @@ static const double dist_table[STS_MAX_CARDINALITY - 1][STS_MAX_CARDINALITY][STS
     }
 };
 
-static sts_symbol get_symbol(double value, unsigned int c) {
+static sts_symbol _get_symbol(double value, unsigned int c) {
```
Comment (trink) [inline src/symtseries.c:218]: Why the change to an underscore identifier?

Comment (spolakh) [inline src/symtseries.c:218]: This way static methods are just more easily distinguishable for me (yeah, even though we have sts_\* for non-static), can revert back easily though.
